csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1888,7 +1888,16 @@ Engine version (`version`) of value type `string` with 1 or more characters cont
 
 ##### 3.2.1.12.4 Document Property - Tracking - ID
 
-Unique identifier for the document (`id`) of value type `string` with 1 or more characters holds the Identifier.
+Unique identifier for the document (`id`) of value type `string` with 1 or more characters with `pattern` (regular expression):
+
+```
+    ^[\\S](.*[\\S])?$
+```
+
+Unique identifier for the document holds the Identifier.
+
+> It SHALL NOT start or end with a white space and SHALL NOT contain a line break.
```

Comment:
Let us remove any space coloring :wink: ... end with space characters ...

Why anybody would want to use identifiers with spaces anywhere is a miracle to me.

Reply:
One of our examples does: `Example Company - 2019-YH3234`

I'm happy to change it if the TC wants to prohibit space characters in the `/document/tracking/id`.